docs/typescript/troubleshooting.md

```diff
@@ -204,5 +204,12 @@ Error: 4 DEADLINE_EXCEEDED: context deadline exceeded
 ```
 
 This happens for a number of reasons: Network hiccups, timeouts that are too short, server overloaded.
+Querying a workflow whose query handler causes an error can result in the query call timing out.
```

Comment:
Is this troubleshooting content specific to TS?

Reply:
Nope, possible for anything